app/src/main/java/org/oppia/android/app/spotlight/SpotlightManager.kt

```diff
@@ -0,0 +1,36 @@
+package org.oppia.android.app.spotlight
+
+/** Manager for the [SpotlightFragment]. */
```

Comment:
```suggestion
/** Manager for showing spotlights. */
```

The fragment is one example implementation, so using that as part of the explanation is circular logic.

Reply:
done